docker-compose.test.yml

```diff
@@ -18,10 +20,14 @@ services:
     restart: unless-stopped
     volumes:
       - .:/app
+      - /app/node_modules
     ports:
       - '9324:9324'
     environment:
-      - NEXT_PUBLIC_PRODUCTION_API_URL=http://prod:3001
+      - NEXT_PUBLIC_PROD_HOST_API_URL=http://prod:3001
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://${LOCAL_HOST}:${PLAYWRIGHT_TEST_PORT}"]
+
```

Comment:
Please remove the new line there and be consistent in the code sttyle through all docker compose containers